main/src/main/java/org/mobilitydata/gtfsvalidator/validator/FrequencyTimeInOrderValidator.java

```diff
@@ -20,15 +20,19 @@
 
 import org.mobilitydata.gtfsvalidator.annotation.GtfsValidator;
 import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.StartAndEndTimeEqualNotice;
 import org.mobilitydata.gtfsvalidator.notice.StartAndEndTimeOutOfOrderNotice;
 import org.mobilitydata.gtfsvalidator.table.GtfsFrequency;
 import org.mobilitydata.gtfsvalidator.type.GtfsTime;
 
 /**
  * Validates `frequencies.start_time` is before or equal to `frequencies.end_time` for a single
- * entity.
+ * entity. Does not allow `frequencies.start_time` and `frequencies.end_time` to be equal.
```

Comment:
Instead of "does not allow" I would say something like "validates if", as it's a warning and not an error. It would be good to use more general language like this for all validation notices so we don't have to go back and update Javadocs if the severity level changes.